src/native_driver.jl

```diff
@@ -10,6 +10,32 @@ const empty_pod = """{
     }
 }"""
 
+
+"""
+    default_namespace() -> String
+
+Determine the default Kubernetes namespace as specified by the current `kubectl` context.
+Typically, the default namespace is: "default".
+
+If the namespace isn't set, the current context isn't set, or the current context isn't
+defined then an empty string will be returned.
```

Comment:
My understanding is that the default namespace is called `default`, regardless of which namespace is active according to the current context. If no context is set (or if the current context doesn't specify a namespace), then `default` _is_ the active namespace. Would it make more sense to return `default` unless otherwise specified?

Why not call this function `current_namespace` or `active_namespace` to avoid confusion with the actual default namespace (i.e. the one called `default`)?

Reply:
Completely sensible suggestion. I named this function prior to getting into these details. I think `current_namespace` is good as it mirrors the "current-context" terminology seen in `kubectl config --help`

Reply:
> My understanding is that the `default` namespace is called default, regardless of which namespace is active according to the current context

Do you have any references for this? I originally thought this too

Reply:
I ran a quick experiment to verify the behaviour. Essentially, I just unset the namespace for minikube:
```
kubectl config set-context minikube --namespace=""
```
And then created a new deployment and checked what namespace it was included in: "default". I'll do a quick update here then to return "default" if not explicitly set.

Thanks for the feedback

Reply:
I'll note I'm rather surprised that `kubectl config view` doesn't return `namespace: "default"` when the namespace isn't defined for a context. It already seems to auto-populate `user` and `cluster` when they aren't defined.